qwazzock/server.py

```diff
@@ -21,6 +22,11 @@ def player_client():
     def host():
         return render_template("host_client.html", page_name="host_client")
 
+    @app.route("/static/questions/<string:requested_image>")
+    def send_question_image(requested_image):
+        logger.info(f"Sending question image {requested_image} from content directory.")
+        return send_file(f"{game.content_path}/questions/{requested_image}")
```

Comment:
I don't know enough about Python to tell whether this opens us up to path navigation, but I haven't been able to prove it yet.